profile/Readme.md

```diff
@@ -5,55 +5,26 @@
+  </a>
+  <a href="https://discord.gg/3tNUBWQRXv">
+    <img alt="Discord" src="https://img.shields.io/discord/1179611100183011429?style=flat&logo=discord&logoColor=%23FFFFFF&label=discord&labelColor=%235865F2&color=23a55a">
+  </a>
   <a href="https://github.com/TKMM-Team/Tkmm">
-    <img src="https://img.shields.io/github/stars/TKMM-Team/Tkmm?style=for-the-badge&logoColor=FFCB41&color=FFCB41&labelColor=2A2C33&logo=github" alt="Stars"/>
+    <img alt="GitHub Repo stars" src="https://img.shields.io/github/stars/TKMM-Team/Tkmm?style=flat&logo=data%3Aimage%2Fsvg%2Bxml%3Bbase64%2CPHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjBweCIgdmlld0JveD0iMCAtOTYwIDk2MCA5NjAiIHdpZHRoPSIyMHB4IiBmaWxsPSIjMDAwMDAwIj48cGF0aCBkPSJtNDgwLTI4NS0xNjQgOThxLTExIDYtMjEuNSA1dC0xOC41LTdxLTgtNi0xMi0xNi41dC0xLTIxLjVsNDMtMTgzLTE0NS0xMjNxLTktOC0xMS0xOC41dDEtMjAuNXEzLTEwIDExLTE2LjV0MjAtNy41bDE5MC0xNyA3NS0xNzRxNS0xMSAxNC0xNi41dDE5LTUuNXExMCAwIDE5IDUuNXQxNCAxNi41bDc1IDE3NSAxOTAgMTZxMTIgMSAyMCA4dDExIDE3cTMgMTAgLjUgMjBUNzk4LTUzM0w2NTQtNDEwbDQzIDE4M3EzIDExLTEgMjEuNVQ2ODQtMTg5cS04IDYtMTguNSA3dC0yMS41LTVsLTE2NC05OFoiLz48L3N2Zz4%3D&labelColor=%23FFFFFF&color=%23FBC84F">
   </a>
-</p>
 
-[![]("https://gamebanana.com/wips/embeddables/81203?type=medium")](https://gamebanana.com/wips/81203)
+  <br />
+  <br />
 
-**TKMM**, (**T**ears of the **K**ingdom **M**od **M**anager) is a mod packager and merger. TKMM automatically takes modded game files and generates bite-sized changelogs for important mod assets (like BYML files, GameDataList, and RSDB) and packages them (along with all other mod assets, like models and textures) into a single archive. A series of those archives can then be merged and combined, in order of preferred priority, into a single mod folder, ready to be loaded into the game.
-
-TKMM smartly combines your mods and maximizes the likelihood that your game will load with all of your favorites ready to go.
-
-<p>
-  <a href="https://github.com/TKMM-Team/Tkmm/issues">
-    <img src="https://img.shields.io/github/issues/TKMM-Team/Tkmm?logoColor=red&color=red&logo=github&style=flat&labelColor=2A2C33" alt="Issues"/>
-  </a> &nbsp;
-  <a href="https://github.com/TKMM-Team/Tkmm/pulls">
-    <img src="https://img.shields.io/github/issues-pr/TKMM-Team/Tkmm?style=flat&labelColor=2A2C33&logoColor=blue&color=blue&logo=github" alt="Open Pull Requests"/>
-  </a> &nbsp;
-  <a href="https://github.com/TKMM-Team/Tkmm/pulls">
-    <img src="https://img.shields.io/github/issues-pr-closed/TKMM-Team/Tkmm?style=flat&labelColor=2A2C33&logoColor=5751FF&color=5751FF&logo=github" alt="Closed Pull Requests"/>
-  </a> &nbsp;
-  <a href="https://totkmods.github.io/tkmm/downloads/">
-    <img src="https://img.shields.io/github/downloads/TKMM-Team/Tkmm/total?style=flat&labelColor=2A2C33&logoColor=008000&color=008000&logo=github" alt="Downloads"/>
-  </a>
```

Comment:
Why remove this?

Reply:
https://github.com/TKMM-Team/.github/pull/2#discussion_r1749001694